package.json

```diff
@@ -48,7 +48,7 @@
     "create-news-verification-hashes": "node utils/create-news-verification-hashes/index.js",
     "lockfile:check": "node utils/lockfile-checker/index.js --check",
     "lockfile:fix": "node utils/lockfile-checker/index.js --fix",
-    "postinstall": "npx tsdx build --tsconfig ./node_modules/cardano-launcher/tsconfig.json"
+    "postinstall": "cd node_modules/cardano-launcher && npx tsdx build --tsconfig tsconfig.json"
```

Comment:
Is this required? If so, you can shorten the command to simply `npx tsdx build`

Reply:
@rhyslbw the original command did not produce any output and the second one did within `node_modules/cardano-launchar/dist/`. However, now I tried running `yarn nix:dev` again on a clean state and this is an error which I get:
<img width="1032" alt="Screenshot 2020-03-13 at 08 19 48" src="https://user-images.githubusercontent.com/376611/76598564-6b6f8780-6503-11ea-8691-22bf4f139288.png">


So it looks like we have either:
A) Original postinstall which "works" but produces no output
B) My 2 commands which somehow worked yesterday but fail today

Reply:
https://github.com/input-output-hk/daedalus/pull/1886/commits/0e2593721b9205d5b0375e54da brings in a new version of `cardano-launcher` that deals with what appears to be a compatibility issue with yarn not triggering the `prepare` script. The added dependencies are the trade-off, but they will go away when we change to use the published package

Reply:
Postinstall hook has been removed.